flink-doris-connector/src/main/java/org/apache/doris/flink/sink/copy/BatchStageLoad.java

```diff
@@ -96,7 +96,9 @@ public BatchStageLoad(
         this.password = dorisOptions.getPassword();
         this.loadProps = executionOptions.getStreamLoadProp();
         this.labelGenerator = labelGenerator;
-        this.hostPort = dorisOptions.getFenodes();
```

Comment:
This one seems particularly odd - isn't fenodes comma speparated string? I did this. thoug, as it is later being prepended to UPLOAD_URL_PATTERN